CONTRIBUTING.md

```diff
@@ -22,15 +20,15 @@ Contributing
 6. Update your branch
   
   ```
-  git remote add upstream https://github.com/highfidelity/hifi
-  git pull upstream master
+  git remote add upstream https://github.com/kasenvr/project-athena
+  git pull upstream kasen/core
   ```
   
   Resolve any conflicts that arise with this step.
 7. Push to your fork
```

Comment:
Fix the display of step 7: when the document is viewed, step 7 doesn't on its own line.